### ISSUE TITLE
Running on Linux

### ISSUE THREAD

**Issue by domluther** [jmichas/PlexTrainingVideoScannerAndAgent#11]
I downloaded this and set this up on Linux. I set it running and it didn't work. This was for one very simple reason - the loggingPath is hardcoded to some random folder on a P: drive. Of course my Linux server does not indeed have a P: drive and so it failed.  A quick edit to LOC_LIN and loggingPath and I have a Lynda library on Plex.

**Comment by danielporto** [comment]
I guess it was fixed. I've just installed on QNAP and works nicely. Thanks

**Comment by pibi73** [comment]
> I downloaded this and set this up on Linux. I set it running and it didn't work. This was for one very simple reason - the loggingPath is hardcoded to some random folder on a P: drive. Of course my Linux server does not indeed have a P: drive and so it failed. A quick edit to LOC_LIN and loggingPath and I have a Lynda library on Plex.

Hi, do you can pls copy your configuration ? 
Thanks a lot and best regards.

**Comment by jmichas** [comment]
Yeah, I realize this is an issue, but way back when I wrote it there was like zero documentation for writing the plugins. I could not figure out how to make the logging path a setting that could be entered in the UI. Anyway, its been forever since I even looked at the code or docs for plugins so I am not sure what is out there.
